src/plugins/intel_gpu/src/graph/program.cpp

```diff
@@ -153,6 +155,8 @@ program::program(engine& engine_ref,
       _compilation_context(compilation_context) {
     _config.apply_user_properties(_engine.get_device_info());
     init_primitives();
+    GPU_DEBUG_INFO << "OpenVINO v" << IE_VERSION_MAJOR << "." << IE_VERSION_MINOR << "." << IE_VERSION_PATCH
+                   << " commit " << OPENVINO_GIT_HASH << std::endl;
     GPU_DEBUG_INFO << "Program config\n" << config.to_string();
```

Comment:
Can't we use core.get_versions() ? as I remember it had the hash.

Reply:
Oh, I didn't know it, much nicer. Thanks telling me.